submissions/EmmanuelIdiaghe/php-database/view.php

```diff
@@ -0,0 +1,64 @@
+<?php
+include_once 'connect.php';
+
+#Variable Declaration
+$notCustomer = !(isset($_COOKIE["details"])); //cookie not set
+$customer_id = ($_GET['view']);
+$info;
+
+
+# Function that sets cookie
+function loadCustomer() {
+    global $pdo;
+    global $customer_id;
```

Comment:
`$customer_id` is already global scoped and can easily be accessed within this function. You don't have to use the `global` keyword